docs/guides/es2k/build_p4_artifacts.md

```diff
@@ -0,0 +1,92 @@
+# Guide to build P4 artifacts
+
+- [1. Overview](#1-overview)
+- [2. Install p4c, cpt and p4-sde RPMs](#2-install-p4c-cpt-and-p4-sde-rpms)
+- [3. Generate P4 artifacts](#3-generate-p4-artifacts)
+- [4. Generate forwarding pipeline binary](#5-generate-forwarding-pipeline-binary)
+   - [4.1 Prepare configuration files](#4-1-prepare-configuration-files)
+   - [4.2 Generate binary](#4-2-generate-binary)
+
+## 1. Overview
+This document explains how to install Intel® IPU E2100 P4 Compiler `p4c-pna-xxp` for Intel® Configurable Pipeline Tool `cpt` to build P4 artifacts. It also explains how to use `tdi_pipeline_builder` utility to generate a binary to program forwarding pipeline config using P4Runtime client `p4rt-ctl`
+
+## 2. Install p4c, cpt and p4-sde RPMs
+Install the `cpt`, `p4c` and `p4-sde` using prebuilt RPMs on x86_64 server available as part of release package. Install using the following commands:
+
+  ```bash
+  # Extract the tarball 
+  tar xvzf mev-hw-b0-<release>-fedora37.tgz
+  cd host/packages/x86_64/
+
+  # Install RPMs from tarball
+  dnf localinstall cpt*.rpm --allowerasing
+  dnf localinstall p4c*.rpm --allowerasing
+  dnf localinstall p4-sde*.rpm --allowerasing
+  
+  ```
+  - Note: Default location for installed packages is `/usr/bin` and these rpms are not relocatable
```

Comment:
default location is "/usr/', 
under which bin or local/bin, lib or local/lib is available